src/EA.Weee.Web/ViewModels/Shared/ViewEvidenceNoteViewModel.cs

```diff
@@ -19,7 +19,9 @@ public class ViewEvidenceNoteViewModel : EvidenceNoteViewModel
         public string WasteDisplay => WasteTypeValue.HasValue ? WasteTypeValue.Value.ToDisplayString() : string.Empty;
 
         [DisplayName("Compliance year")]
-        public string ComplianceYearDisplay => StartDate.Value.Year.ToString();
+        public string ComplianceYearDisplay => ComplianceYear.ToString();
```

Comment:
in the ViewEvidenceNoteViewModelTest.cs I cant see any test referring to that field, would you mind adding some in there? for the display attribute, if it's null, or what will be displayed if it has a value and etc.

Reply:
there is one for the DisplayNameAttribute actually for the compliance year, but add for the other cases